src/main/java/net/rptools/maptool/client/functions/CallFunction.java

```diff
@@ -0,0 +1,171 @@
+/*
+ * This software Copyright by the RPTools.net development team, and
+ * licensed under the Affero GPL Version 3 or, at your option, any later
+ * version.
+ *
+ * MapTool Source Code is distributed in the hope that it will be
+ * useful, but WITHOUT ANY WARRANTY; without even the implied warranty
+ * of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *
+ * You should have received a copy of the GNU Affero General Public
+ * License * along with this source Code.  If not, please visit
+ * <http://www.gnu.org/licenses/> and specifically the Affero license
+ * text at <http://www.gnu.org/licenses/agpl.html>.
+ */
+package net.rptools.maptool.client.functions;
+
+import com.google.gson.JsonArray;
+import com.google.gson.JsonElement;
+import java.awt.EventQueue;
+import java.util.List;
+import net.rptools.maptool.client.MapTool;
+import net.rptools.maptool.client.MapToolLineParser;
+import net.rptools.maptool.client.MapToolVariableResolver;
+import net.rptools.maptool.client.functions.json.JSONMacroFunctions;
+import net.rptools.maptool.client.functions.json.JsonArrayFunctions;
+import net.rptools.maptool.language.I18N;
+import net.rptools.maptool.model.Token;
+import net.rptools.maptool.util.FunctionUtil;
+import net.rptools.parser.Parser;
+import net.rptools.parser.ParserException;
+import net.rptools.parser.VariableResolver;
+import net.rptools.parser.function.AbstractFunction;
+import net.rptools.parser.function.Function;
+
+/**
+ * Represents a callable function within the MapTool client. This class provides the implementation
+ * for executing specific functions as part of the client's functionality.
+ */
+public class CallFunction extends AbstractFunction {
+
+  /** The {@link JSONMacroFunctions} instance used for conversions. */
+  private final JSONMacroFunctions jsonMacroFunctions = JSONMacroFunctions.getInstance();
+
+  /** The {@link JsonArrayFunctions} instance used for conversions. */
+  private final JsonArrayFunctions jsonArrayFunctions = jsonMacroFunctions.getJsonArrayFunctions();
+
+  /** Create the Object and register the paramater count and function name. */
+  public CallFunction() {
+    super(
+        1,
+        UNLIMITED_PARAMETERS,
+        "call",
+        "call.a",
+        "dcall",
```

Comment:
`call.deferred` would be more consistent and discoverable compared to `dcall`.

Reply:
Changed, I am forwarding all hate mail I get for making the function name too long to you ;)